use-cases/point-of-sale.md

```diff
@@ -2,29 +2,27 @@
 
 In Mews charges are not sent to rooms or reservations but the customer profile that are active and attached to a reservation (checked in) or profiles that are classified as `Paymaster`. 
 
-The items that are sent from the POS to Mews are called Orders.
+In Mews, all charges are posted directly to the billing tab of either customer profiles that are active and attached to a reservation (checked in) or profiles that are classified as `Paymaster`. The items that are sent from the POS to Mews are called Orders.
```

Comment:
CC @czech-man the "paymasters" are just a hack and it causes issues to admin team. We can keep it there but it is something that should be solved better.